src/PhyloTree.tsx

```diff
@@ -0,0 +1,183 @@
+import React, {useCallback, useState, useMemo, useEffect} from 'react';
+import type {
+    DataDeep,
+    Datum,
+    PartialTrack,
+    Track,
+    View
+} from 'gosling.js/dist/src/gosling-schema';
+import {GoslingSpec} from "gosling.js";
+import {Vega} from "react-vega";
+
+
+export type PhyloTreeSpec = {
+    type: "tree",
+    data: DataDeep;
+}
+
+interface PhyloTreeProps {
+    data?: Datum[];
+    gosSpec: GoslingSpec;
+    setGoslingSpec: (object) => void;
+    linkedTrack: string;
```

Comment:
this might be clearer

```suggestion
    linkedTrackId: string;
```